### ISSUE TITLE
3.2.3 Consistent Navigation word substitution language is incomplete

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#686]
Two problems:
- (Error from 2013 WCAG2ICT) Currently the word replacement language for 3.2.3 is incomplete because it missed the word replacement for "pages". It currently says: 
>This applies directly as written and described in [Intent from Understanding Success Criterion 3.2.3](https://www.w3.org/WAI/WCAG22/Understanding/consistent-navigation#intent), replacing “set of web pages” with “set of non-web documents” and “set of software programs”.
- the grammar is also a bit off once word replacements are there and read, "on multiple non-web documents" so the replacement should be "in multiple non-web documents" and "in multiple non-web software programs".

So the SC language for the non-web documents and non-web software needs a little tweak AND the word replacement description needs to change as well.

**Comment by maryjom** [comment]
This has been fixed in PR #687 and can be closed.
